docs/migrations/web-react/MIGRATION-v3.md

```diff
@@ -88,6 +88,21 @@ Manually replace the props in your project.
 
 ---
 
+### Link `hasVisitedStyleAllowed`
+
+The `hasVisitedStyleAllowed` property was added. This property allows the link to have visited state style. There is also a change in the use of the visited state style for the `Heading` component, which had previously this state set by default.
+Now a prop `hasVisitedStyleAllowed` has to be added to enable the visited state.
+
+#### Migration Guide
+
+If you want to enable the visited state style for the `Link` component, please manually add the `hasVisitedStyleAllowed` prop.
+
+<details>
+  <summary>🔧 Manual Migration Steps</summary>
```

Comment:
if it would be possible to create a codemod, it would be great.
It should add the prop to all links inside the Heading component (no matter how deep)

Reply:
codemods added